### PR TITLE
COMMERCE-11938 - Channel notification To field has a limit of 75 characters

### PR DESCRIPTION
**Pull request by victorcdp**
https://liferay.atlassian.net/browse/COMMERCE-11938

@andrea-ale-sbarra Can you review this one? Tied to an LPP. Thank you.

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

**Comment by victorcdp** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-notification-service/src/main/resources/META-INF/portlet-model-hints.xml [hunk 1]
```diff
@@ -64,17 +64,20 @@
...
 		</field>
 		<field name="from" type="String">
+			<hint-collection name="CLOB" />
 			<validator name="required" />
 		</field>
 		<field localized="true" name="fromName" type="String">
 			<validator name="required" />
 		</field>
-		<field name="to" type="String" />
+		<field name="to" type="String">
```
[the remark this answers is not in the document]
Comment (victorcdp) [reply modules/apps/commerce/commerce-notification-service/src/main/resources/META-INF/portlet-model-hints.xml:73]: @andrea-ale-sbarra I think enlarging the column by a little bit would just postpone the issue, sooner or later some client would reach the new limit and the issue would come up again. Also, the default liferay Notification implementation uses longText, so i figured Text field would be a good choice for this one.

[the remark this answers is not in the document]
Comment (luca-pellizzon) [reply modules/apps/commerce/commerce-notification-service/src/main/resources/META-INF/portlet-model-hints.xml:73]: it's an almost dead functionality, so as long as it is managed properly this solution is good to me

**Review by luca-pellizzon — APPROVED** [review]

File: modules/apps/commerce/commerce-notification-service/src/main/resources/META-INF/portlet-model-hints.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-notification-service/src/main/resources/META-INF/portlet-model-hints.xml:73]: ok then, thank both!

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by Brunno0liveira** [comment]
Resent at https://github.com/liferay-commerce/liferay-portal/pull/4011

Thanks!

**Comment by Brunno0liveira** [comment]
ci:close

**Pull request closed without merging**